Controller/PlatformUIController.php

```diff
@@ -33,6 +44,33 @@ public function shellAction()
         );
     }
 
+    /**
+     * Load JS or CSS assets.
+     *
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function combineLoaderAction(Request $request)
+    {
+        $files = array_keys($request->query->all());
+
+        try {
+            $content = $this->loader->getCombinedFileContent($files);
+            $type = $this->loader->getCombinedFileContentType($files);
```

Comment:
I'd actually get the types first, and the contents after that: if there is an error in the types,